package/gluon-status-page/javascript/status-page.js

```diff
@@ -192,8 +192,8 @@
 
 		div.style.display = '';
 		var table = document.getElementById('mesh-vpn-peers');
```

Comment:
We can avoid repeating `firstElementChild` a lot in the JS code by moving the `"mesh-vpn-peers"` ID from the `<table>` to the `<tbody>`.

Reply:
same here, i think that the following could be clearer:

```javascript
var table = document.getElementById('mesh-vpn-peers');
var tbody = table.firstElementChild;
while (tbody.lastChild)
	tbody.removeChild(tbody.lastChild);
```

Reply:
I don't see how that would be clearer than
```js
var tbody = document.getElementById('mesh-vpn-peers');
while (tbody.lastChild)
	tbody.removeChild(tbody.lastChild);
```
(with the ID moved in the HTML)

Reply:
Updated it to use id on the element so its basically a naming change